packages/agent/src/agents/operators/__tests__/modelSelector.spec.ts

```
@@ -564,36 +550,33 @@ describe('ModelSelector', () => {
 
-        const mockChatOpenAI = jest.fn().mockImplementation(() => {
-          throw new Error('Model creation failed');
-        });
-        jest
-          .mocked(require('@langchain/openai').ChatOpenAI)
-          .mockImplementation(mockChatOpenAI);
+      const mockChatOpenAI = jest.fn().mockImplementation(() => {
+        throw new Error('Model creation failed');
+      });
+      jest
+        .mocked(require('@langchain/openai').ChatOpenAI)
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)